decidim-core/spec/models/decidim/messaging/message_spec.rb

```diff
@@ -3,20 +3,47 @@
 require "spec_helper"
 
 describe Decidim::Messaging::Message do
-  context "when sender is not a participant" do
-    subject do
-      conversation = Decidim::Messaging::Conversation.start(
-        originator: originator,
-        interlocutors: [recipient],
-        body: "Hei!"
-      )
-
-      conversation.add_message(sender: create(:user), body: "sneaking in")
+  describe "#sanitized_body" do
+    let!(:originator) { create(:user) }
+    let!(:recipient) { create(:user) }
+
+    context "when sender is not a participant" do
```

Comment:
what are we testing here?

Reply:
I didn't add the test, I moved it inside the same describe by accident and it counts as an addition.